xcfs/Package.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+_ = Package(
+    name: "deps",
+    platforms: [.macOS("11")],
```

Comment:
Is it okay to drop support for macOS Catalina or earlier as a build environment?